### PR TITLE
Remove Misc class

### PR DESCRIPTION
**Pull request by kamil-tekiela**
Follow up from #451 but this time I move the class to `SelectStatement` where it belongs due to feature envy.

### CODE REVIEW

File: src/Statements/SelectStatement.php [hunk 1]
```diff
@@ -366,4 +366,85 @@ public function getClauses()
...
+            if (! isset($tables[$thisDb])) {
+                $tables[$thisDb] = [];
+            }
+
+            $tables[$thisDb][$expr->alias] = $expr->table;
+        }
+
+        foreach ($this->expr as $expr) {
+            if (! isset($expr->column, $expr->alias) || ($expr->column === '') || ($expr->alias === '')) {
+                continue;
```
Comment (williamdes) [inline src/Statements/SelectStatement.php:432]: can you catch this mutant ?

**Review by williamdes — APPROVED** [review]
Very nice idea !

File: src/Statements/SelectStatement.php (hunk 1 above)
Comment (kamil-tekiela) [reply src/Statements/SelectStatement.php:432]: I don't know, I haven't analysed what this method does yet. O just copied and pasted the whole method.

Comment (williamdes) [reply src/Statements/SelectStatement.php:432]: Okay, let's merge this one as it is

**Review by williamdes — APPROVED** [review]

**Review by williamdes — APPROVED** [review]

**Pull request merged**